### PR TITLE
vote: gozssky as TiKV reviwer

### PR DESCRIPTION
**Pull request by Little-Wallace**
vote: gozssky as TiKV reviwer

### CODE REVIEW

File: votes/0141-gozssky-as-tikv-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+## Proposal
+
+[@gozssky](https://github.com/gozssky) has been working on `tikv/tikv` for one years. He has continuously contributed to sst_importer and raftstore.
+
+* [Authored pull requests](https://github.com/tikv/tikv/commits?author=gozssky).
+* [Reviewed pull requests](https://github.com/tikv/tikv/pulls?q=is%3Apr+reviewed-by%3Agozssky)
+
+I (@Little-Wallace) hereby nominate @gozssky as TiKV Reviwer and call for a vote.
+
+## Result
```
Comment (tisonkun) [inline votes/0141-gozssky-as-tikv-reviewer.md:12]:
```suggestion
## Deadline

The vote will be open for at least 72 hours unless there is an objection or not enough votes.

## Result
```

according to #143 .

Also you can update the filename to `0143-...`

**Comment by tisonkun** [comment]
/cc @hicqu @BusyJay 

You may be interested in this vote.

**Review by skyzh — APPROVED** [review]
+1 binding, congrats!

**Review by BusyJay — APPROVED** [review]
Thanks!

**Comment by tisonkun** [comment]
/assign @BusyJay @skyzh 

We already over the minimal length requirement and with 2 binding votes. Could you please conclude the result and merge this vote?

**Comment by skyzh** [comment]
Congrats @gozssky! You are now a TiKV reviewer!

@tisonkun would you please help add them to the TiKV community team?

**Comment by tisonkun** [comment]
@skyzh my please...and done.

**Pull request merged**